service-spec/schema-validator/src/main/resources/hprof/0.6.4/profile-spec.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
This file should not come in the review. Remove this and commit. Same for hspec file